nrf_security/doc/driver_config.rst

```diff
@@ -403,3 +403,22 @@ You can use the following PSA driver-specific configurations for fine-grained co
 +-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
 | SHA-512               |  Not supported                                                | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_512_OBERON` |
 +-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
+
+Password-authenticated key agreement configurations
+===================================================
+
+To enable Password-authenticated key agreement support, set one or more of the following Kconfig options:
+
++-----------------------+-----------------------------------------------+
+| PAKE algorithm        | Configuration option                          |
++=======================+===============================================+
+| EC J-PAKE             | :kconfig:option:`CONFIG_PSA_WANT_ALG_JPAKE`   |
++-----------------------+-----------------------------------------------+
+| SPAKE2+               | :kconfig:option:`CONFIG_PSA_WANT_ALG_SPAKE2P` |
++-----------------------+-----------------------------------------------+
+| SRP                   | :kconfig:option:`CONFIG_PSA_WANT_ALG_SRP`     |
++-----------------------+-----------------------------------------------+
+
+.. note::
```

Comment:
We should follow the pattern of having another subsection for driver configuration:

Password-authenticated key agreement driver configurations
And list the driver support.